src/components/Body.css

```diff
@@ -0,0 +1,14 @@
+.presentation{
```

Comment:
Formatting:
space before open bracket,
tabs in .title class

Use auto formatter